custom_components/tuya_local/devices/techinc_smart_3m_neon_strip.yaml

```diff
@@ -0,0 +1,187 @@
+name: Tech Inc 3m neon strip
+products:
+  - id: szzsfdqcrqkczzfi
+    name: TI 3M neon light strip
```

Comment:
TI usually is Texas Instruments, and 3M is also a brand name unrelated to this. Suggest using "Tech Inc 3m neon" (light strip is already covered in the top level name)

Reply:
Changed